src/main/java/ch/njol/skript/conditions/CondIsEnchanted.java

```diff
@@ -34,49 +35,53 @@
 import ch.njol.skript.util.EnchantmentType;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Is Enchanted")
 @Description("Checks whether an item is enchanted.")
 @Examples({"tool of the player is enchanted with efficiency 2",
-		"helm, chestplate, leggings or boots are enchanted"})
-@Since("1.4.6")
+		"helm, chestplate, leggings or boots are enchanted",
+		"",
+		"# For enchanted books",
+		"player's tool is stored enchanted",
+		"event-item is enchanted with stored power 2"})
+@Since("1.4.6, INSERT VERSION (stored enchantments)")
 public class CondIsEnchanted extends Condition {
```

Comment:
It may be better to make a separate condition for stored enchantments.
Syntax could be:
```
%enchantmenttypes% (is|are|is not|isn't|are not|aren't) stored on %itemtypes%
%itemstypes% (has|have|doesn't have|don't have) %enchantmenttypes% stored (maybe?)
```